Kitodo/src/test/java/org/kitodo/services/workflow/WorkflowControllerServiceTest.java

```diff
@@ -22,12 +22,16 @@
 
 public class WorkflowControllerServiceTest {
 
-    private static final WorkflowControllerService workflowControllerService = new ServiceManager().getWorkflowControllerService();
+    private static WorkflowControllerService workflowControllerService;
 
     @BeforeClass
     public static void prepareUser() {
-        User user = new User();
-        workflowControllerService.setUser(user);
+        try {
+            workflowControllerService = new ServiceManager().getWorkflowControllerService();
+        } catch (Exception e) {
+            User user = new User();
```

Comment:
Is setting up a new empty user object correct solution in any case an exception is thrown on initialising the test suite?

Reply:
Ok, but I see here a second issue: if an exception is thrown how did you know that `workflowControllerService` is correct initialised and usable?

Reply:
Good point, I should change to this certain exception.